### ISSUE TITLE
Only Host associated object syncs Instance Variables, Peer associated objects do not sync Instance Variables.

### ISSUE THREAD

**Issue by Alderin1974** [Scirra/Construct-3-bugs#4228]
## Problem description

The Host and Peers use a Player object synced with the Multiplayer object. Instance Variables that are only changed on the Host don't sync to the Peers, except when they are changed on the Host's associated Player object.

## Attach a .c3p

https://drive.google.com/file/d/1JWZ9fIobtQ5Vb1OzfTGNin320KvQxj1o/view?usp=sharing

## Steps to reproduce

1. Open project in a browser, enter a username, and log in.
2. Observe the Host firing pattern where the weapon charges up before firing, and cools down after firing.
3. Open project in a separate browser to connect as a Peer, log in with a different username.

## Observed result

1. When using the Peer screen: Observe no charging or cool down when firing on the Peer display, but Observe the charging and cool down properly displayed on the Host display.
2. When using the Host screen: Charging and cool down effects display properly.

Alternatively, run both Host and Peer in Debug mode, and view the Player object's Instance Variables. Note that the Host's Player object on the Peer has the "warm" and "cool" Instance variables changed, but the Peer's Player object does not, even though they change on the Host.

## Expected result

The "warm" and "cool" instance variables on the Player object that drive the charging and cool down display should sync from the Host to the Peer(s), allowing charging and cool down to properly display on the Peer(s).

## More details

**Affected browsers/platforms:**
Tested on Ubuntu Linux 18.04LTS, Chrome Version 85.0.4183.102 (Official Build), and on Firefox 80.0.1 (64-bit). Tested with both C3r210.2 and C3r215. 

**First affected release:** r210.2

## System details

<details><summary>View details</summary>

Platform information
Browser: Chrome
Browser version: 85.0.4183.102
Browser engine: Chromium
Browser architecture: (unknown)
Context: webapp
Operating system: Linux
Operating system version: (unknown)
Operating system architecture: 64-bit
Device type: desktop
Device pixel ratio: 1
Logical CPU cores: 8
Approx. device memory: 8 GB
User agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.102 Safari/537.36
C3 release: r210.2 (stable)
Language setting: en-US

Local storage
Storage quota (approx): 541 gb
Storage usage (approx): 160 mb (0%)
Persistant storage: Yes

Browser support notes
This list contains missing features that are not required, but could improve performance or user experience if supported.

Nothing is missing. Everything is OK!
WebGL information
Version string: WebGL 2.0 (OpenGL ES 3.0 Chromium)
Numeric version: 2
Supports NPOT textures: yes
Supports GPU profiling: yes
Supports highp precision: yes
Vendor: Google Inc.
Renderer: ANGLE (NVIDIA Corporation, GeForce RTX 2060/PCIe/SSE2, OpenGL 4.5 core)
Major performance caveat: no
Maximum texture size: 32768
Point size range: 1 to 2047
Extensions:

EXT_color_buffer_float
EXT_disjoint_timer_query_webgl2
EXT_float_blend
EXT_texture_compression_bptc
EXT_texture_compression_rgtc
EXT_texture_filter_anisotropic
KHR_parallel_shader_compile
OES_texture_float_linear
WEBGL_compressed_texture_etc1
WEBGL_compressed_texture_s3tc
WEBGL_compressed_texture_s3tc_srgb
WEBGL_debug_renderer_info
WEBGL_debug_shaders
WEBGL_lose_context
OVR_multiview2
Audio information
System sample rate: 44100 Hz
Output channels: 2
Output interpretation: speakers
Supported decode formats:

Ogg Opus (audio/ogg; codecs=opus)
WebM Opus (audio/webm; codecs=opus)
Ogg Vorbis (audio/ogg; codecs=vorbis)
WebM Vorbis (audio/webm; codecs=vorbis)
MPEG-4 AAC (audio/mp4; codecs=mp4a.40.5)
MP3 (audio/mpeg)
FLAC (audio/flac)
PCM WAV (audio/wav; codecs=1)
Supported encode formats:

WebM Opus (audio/webm; codecs=opus)
Video information
Supported decode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)
Ogg Theora (video/ogg; codecs=theora)
H.264 (video/mp4; codecs=avc1.42E01E)
Supported encode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)
</details>

**Comment by AshleyScirra** [comment]
The provided link is not public. Please provide a publicly available project file as per the bug report guidelines.

Please also note it's very important to provide only a minimal project demonstrating the issue. It's exceptionally difficult to debug multiplayer games, and it's also common that bug reports are actually just mistakes in events. Your report describes gameplay mechanics - a minimal project should just use the fewest objects and events possible, and e.g. if the problem is to do with instance variables, just display the instance variables with text objects or something similar, so the values can easily be directly observed instead of relying on gameplay mechanics.

**Comment by Alderin1974** [comment]
Sorry about the link permissions. I do understand that the smaller the project is, the easier it is to navigate, for any reason. I did try to strip out everything unrelated while leaving it functional, but I will make a more concise example using the "Pong" example as a base, and will ensure the link is public once uploaded.

**Comment by Alderin1974** [comment]
Ok, in building the example project, I found the issue: I had put "Client value tag" for all of my Instance Variables. When I cleared those from the "warm" and "cool" variables, the system worked as expected. I might suggest an "Only use for Client Input values", or other re-wording of the tooltip. The current wording makes it sound like a good idea to have something there to reduce latency.

Thank you for your quick response, and sorry for the non-bug.
